library/src/main/java/com/bumptech/glide/load/engine/Engine.java

```diff
@@ -293,6 +293,10 @@ public void onResourceReleased(Key cacheKey, EngineResource resource) {
         }
     }
 
+    public void clearDiskCache() {
+        diskCacheProvider.getDiskCache().clear();
```

Comment:
The name implies blocking work. Thing about how would you use it: call `clear` then call `.with.into`, how would you make `clear` finish before checking the cache during processing the request?